Dockerfile

```diff
@@ -20,3 +20,5 @@ RUN apt update \
 
 # copy project
 COPY app/ /usr/src/app/
+
+CMD python index.py --host 0.0.0.0 --port 8050
```

Comment:
Not sur we need this it's already In the docker compose

Reply:
also missing blank line

Reply:
I like to have CMD or entrypoint directly in the Dockerfile to be able to use the container without the docker-compose , is it ok ?

=> I thought ruff would fix the blank line, I am gonna do it

Reply:
yes not problem for new just wanted to be sure that it was made on purpose